### DISCUSSION TITLE
Seasonal Race Re-Structure Proposal

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3895] [Ideas]
A discussion on some community inspired proposals packaged into 1, to restructure the seasonal race. Open to feedback, improvements and criticisms. Some of the core changes that would be applied are listed below and numbered. Do note, I mam proposing for ALL these changes to be applied, not just one or two

### **Core changes**

- Removal of Crop Ticket RNG
- Move or add strong items into Stella's Megastore for the 3rd/last month of season
- Move or add more basic low tier utility items into Stella's Megastore
- Ensure Bert does not ask for anything too outlandish. Alternative is to remove.
- Move to weekly chore structure (currently being built in Faction Chores)

### **1. REMOVAL OF CROP TICKET RNG**
This has been a point of contention for many seasons amongst players. There are some who don't mind it, and some who do. 

**Issue:** A player may go through a period of 3 months, logging in EVERY day, completing EVERY delivery, EVERY chore, buy EVERY Bert item and trade EVERY flower to Poppy... but still has a chance to miss out to someone else in a ticket auction for the best item due to Crop RNG. This makes the gameplay very off-putting, and someone who experiences will think twice about participating as much/at all in future seasons.

**Solution:** I think we should try a season without it. Removing the Crop Ticket RNG mechanic allows us to exactly determine the MAX number of tickets players will earn throughout the season.. 
If we know that by x week a player doing everything should roughly have 3k tickets, we can price HIGH tier items at Stella's store for 2.9k. If a player puts in the work for 3 months, they get that prize, it is entirely up to them. We get to set the required commitment/spend for items.

**Considerations:** Some Seasonal Ticket Auctions would end up in ties if everyone is doing the same tasks. These would be decided by XP. "Wouldn't top XP players win all the time?" They could, but this is taking into consideration that these players are also completing EVERY task during the season. Removing RNG tickets also means that missing 1 chore, or 1 delivery, they are behind the leading pack, forever. To alleviate the concern and impact on this, it comes down to providing items at Stella's Store, mentioned below.

### **2. STRONGER ITEMS IN STELLA'S STORE**
In combination with removing Crop Ticket RNG, we would look to add at least 1 other strong item in Stella's Megastore requiring 95-99% of the tickets available. This would also sit in the same month as the other strong item in the seasonal ticket auction. Note, the strong item in ticket auctions and strong item at stella would be DIFFERENT items.
EG. If the best item in the season is up for ticket auction during the 3rd month, there would also be a really strong item (maybe not AS strong as the auctioned item..) in Stella's Store requiring a high number of tickets. We can set this number because as previously mentioned, we KNOW the max number of tickets players would gather.

Why have a strong item in Stella's store? A player who misses out on the strong item ticket auction due to XP, is able to fall back on getting this item from Stella. A criticism to missing out in the last auctions is that players end up with thousands of tickets with nothing to spend them on, or something usually low tier at Stella. Imagine going through 3 months completing everything available every day and missing out entirely. A strong item being available in the 3rd month of Stella diminishes the anguish of players who sadly missed out even if they collected all the tickets. Keeping players engaged and encouraging them to keep playing is important. It falls on them to complete all the tasks and no longer falls to whether other players got extra luck with crop rng. it becomes more PVE rather than PVP.

### **3. MORE LOW TIER UTILITY ITEMS IN STELLA'S STORE**
There should be more low-tier items at Stella's store with basic utility requiring seasonal tickets to attain. New/low level players who cannot compete for the stronger items may still look to complete chores/deliveries and earn tickets if it means they could acquire some new items. I think having 3 utility items per month at Stella's store would be great, these can be 1-2 low tier items, and 1 mid-strong item.

The low tier items can have basic boosts, as low as +20% pumpkin soup XP or +0.05 wood yield. We need to ensure all players can participate to an extent in the season. A common criticism for the current season is that there really isn't anything for players below Desert Island. Stella's items boost GH items (available at Desert Island) and auctions will be hotly contested. It's fine to have these items, I think it would be better if there was at least ANOTHER option for these players to obtain.

### **4. BERT**
Removing Crop RNG makes Bert very important in terms of tickets.. missing Bert once means being behind the pack. Therefore, Bert should be asking for items/collectibles/wearables high in volume or attainable ingame (landscape items or wardrobe wearables). Alternative is to remove him completely or re-work him.

### **5. WEEKLY CHORES**
Changing the daily chores to weekly is currently being developed for Faction House. This would hopefully move to the seasonal structure if proven successful. As mentioned in other mediums, weekly chores will allow players to continuosly complete chores one after the other, also able to see upcoming chores. This alleviates the timezone issue. Previewing chores the day before the week starts will completely solve the timezone issue.

If we want players to have more differentiation between tickets and not rely on XP ti break the ties, weekly chores could be expanded and more could be added, making it so you could grind/complete chores almost ad infinitum (not limit on chores for the week. Similarly to the earlier seasons.

![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/2eed4f75-2e23-456c-8d55-bae6a67d83d7)

With chores being weekly, if they start on Monday then ticket auctions should happen on the weekend, Saturday/Sunday.

### **Example Item Structure**

Items:
UTILITY
-U1 (basic)[stella]
-U2 (medium)[stella]
-U3 (basic)[stella]
-U4 (medium)[stella]
-U5 (basic)[stella]
-U6 (strong)[stella]
-U7 (basic)[auction]
-U8 (medium)[auction]
-U9 (medium)[auction]
-U10 (strong)[auction]

DECO
-D1[stella]
-D2[stella]
-D3[stella]
-D4[stella]
-D5[stella]
-D6[stella]
-D7[auction]
-D8[auction]
-D9[auction]
-D10[auction]

Assigned:
STELLA MONTH 1
-U1 (basic)
-U2 (medium)
-D1
-D2

STELLA MONTH 2
-U3 (basic)
-U4 (medium)
-D3
-D4

STELLA MONTH 3
-U5 (basic)
-U6 (strong)
-D5
-D6

AUCTIONS FOR SFL, BBS, TICKETS
-U7 (basic)
-U8 (medium)
-U9 (medium)
-U10 (strong, occurring in month 3)
-D7
-D8
-D9
-D10

**Comment by Dionis404** [comment]
**1. Removing the crop ticket RNG mechanic might make the race less exciting.**
We can consider options to modify this mechanic for a more balanced approach. For example, tickets could drop with a certain probability upon completing specific actions, or we could create a system where players earn tickets for achieving certain milestones. This way, we can retain the element of randomness but with more controlled outcomes.


**2. Adding a strong item to Stella's store could motivate more players to complete all deliveries.**
However, it is important to analyze how this will impact the game economy. We need to ensure that such changes do not lead to inflation or a significant imbalance in resource distribution. It might be wise to limit the number of such items or introduce additional requirements for obtaining them to maintain economic stability.

**3. Low-tier items are a great idea.**
I would suggest restricting these items to the level of Bumpkin or farm expansion. This way, new players can receive bonuses for leveling up without higher-level players taking these items for resale. This will enhance accessibility and attractiveness for newcomers, providing them with the necessary resources for development.


**4.Bert is a crucial mechanic in the race for obtaining top items.**
If Bert stops requesting items that were available in previous seasons, the demand for decorative clothing will significantly decrease. It might be better to balance his tasks to include both old and new items, maintaining interest in various aspects of the game and encouraging diversity in item collection.


**5.Yes!**
Switching to weekly chores is a great idea. This change will allow players to complete tasks at their convenience, solving the issue with time zones and providing more equal conditions for all participants. A more flexible schedule for completing chores can also increase player engagement and improve the overall gaming experience.
